doc/source/developer_guide/importers_module_example.py

```diff
@@ -0,0 +1,251 @@
+# -*- coding: utf-8 -*-
+"""
+One line description of this module. E.g. "Awesome Bureau of Composites importers."
+
+Here you can write a more extensive description (optional). For example, describe the
+readers that are implemented (e.g. Grib2 and Netcdf) and any other thing that you
+consider relevant.
+"""
+
+# Here import your libraries
+import numpy as np
+
+### Uncomment the next lines if pyproj is needed for the importer.
+# try:
+#     import pyproj
+#
+#     PYPROJ_IMPORTED = True
+# except ImportError:
+#     PYPROJ_IMPORTED = False
+
+from pysteps.decorators import postprocess_import
+
+
+# Function import_abc_grib to import Grib files from the Awesome Bureau of Composites
+# ===================================================================================
+
+# IMPORTANT: The name of the importer should follow the "importer_institution_format"
+# naming convention, where "institution" is the acronym or short-name of the
+# institution. The "importer_" prefix to the importer name is MANDATORY since it is
+# used by the pysteps interface.
+#
+# Check the pysteps documentation for examples of importers names that follow this
+# convention:
+# https://pysteps.readthedocs.io/en/latest/pysteps_reference/io.html#available-importers
+#
+# The function prototype for the importer's declaration should have the following form:
```

Comment:
What do you mean for function prototype? Can we simplify this sentence with something like "The definition of your importer function should have the following form" ?

Reply:
By prototype, I was referring to the function's form, but it was indeed redundant and confusing. I changed the sentence as you suggested (commit 61052e2).